src/libraries/Native/Unix/System.Native/pal_networkchange.c

```diff
@@ -68,12 +68,9 @@ static NetworkChangeKind ReadNewLinkMessage(struct nlmsghdr* hdr)
     assert(hdr != NULL);
     struct ifinfomsg* ifimsg;
     ifimsg = (struct ifinfomsg*)NLMSG_DATA(hdr);
-    if (ifimsg->ifi_family == AF_INET)
+    if (ifimsg->ifi_family == AF_UNSPEC)
```

Comment:
Checking for AF_INET was simply wrong before?

Reply:
Yes, and on the managed side `LinkAdded` was ignored. So there is no possibility of regression here.

Reply:
My guess that in the past presence of IPv4 address was considered as "available". Perhaps pre-dates IPv6